curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/64649d2459ad94e0a772fc3f.md

```diff
@@ -9,7 +9,7 @@ dashedName: step-95
 
 Now you'll add a top margin to the paragraph element.
 
-Add a `style` attribute to the paragraph element and use string interpolation to set the value to `margin-top: ${currMarginTop}px;`, where `currMarginTop` is the `marginTop` property of the current object.
+Add a `style` attribute to the paragraph element and use string interpolation to set the value to `"margin-top: ${currMarginTop}px;"`, where `currMarginTop` is the `marginTop` property of the current object.
```

Comment:
we are asking the campers to use string interpolation, changing this string to double quote doesn't work in this scenario

Reply:
Hi @RGHANILOO, the double quotes work in this scenario because they are inside the backticks, you can see the line 192 in this file